ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTableTreeEvents.scala

```diff
@@ -126,7 +101,11 @@ private[events] trait EventsTableTreeEvents { this: EventsTable =>
   ): SimpleSql[Row] = {
     val witnessesWhereClause =
       sqlFunctions.arrayIntersectionWhereClause("tree_event_witnesses", requestingParty)
-    SQL"select #$selectColumns, array[$requestingParty] as event_witnesses, case when submitter = $requestingParty then command_id else '' end as command_id from participant_events where transaction_id = $transactionId and #$witnessesWhereClause order by node_index asc"
+    SQL"""select #$selectColumns, array[$requestingParty] as event_witnesses,
+                 case when submitter = $requestingParty then command_id else '' end as command_id
+          from participant_events
+          where transaction_id = $transactionId and #$witnessesWhereClause
+          order by node_index asc"""
```

Comment:
The original intent of the single-line SQL query was for it to be less obtrusive in logs. But was just a bad call on my side, thanks for fixing it.